README.md

```diff
@@ -94,7 +91,8 @@ Steps may differ for different OSes. For example, in Windows, use a backslash `\
 
 ## Settings ⚙
 
-You may change the appropriate settings in the `usr_config.json` file, which is stored in the `usr` folder.
+You may change the appropriate settings in the `config.json` file, which is stored in your OS' default configuration
+folder (.i.e. `$XDG_CONFIG_HOME` on Linux, *BSD, and MacOS; `%LOCALAPPDATA%` for Windows)
```

Comment:
is there a reason BSD comes with an asterisk?

Reply:
It's mainly due to the various iterations of BSD (.e.g. FreeBSD, NetBSD, OpenBSD etc) and this would refer to all variations of such as there is no such distribution called 'BSD'. 

I've seen people refer to BSD as a whole as '*BSD' quite frequently so I deemed it appropriate.

Reply:
ah ok. i thought there was supposed to be a footnote somewhere